src/js/components/TaskListItemComponent.jsx

```diff
@@ -158,6 +158,13 @@ var TaskListItemComponent = React.createClass({
     </a>);
   },
 
+  getDebugLink: function () {
```

Comment:
I would instead add a generic function:
generateHeader() and generateContent() -> both would render the <th> and <td> content.
Thus, we could plug any feature just in the configuration files

Reply:
I agree that generalizing the concept of link generators is a good idea for code reuse and would be feasible but it would require a bit of work since the functionalities I added are not all at the same level and affect different components (app level components, task level components and it could be anything else in the future).

Moreover, I'm a bit worried about making this runtimeConfig smarter. I don't really want it to become an intelligent application itself. It would most probably open the door to untested code.